### ISSUE TITLE
[Bug] Cross references from data section not showing

### ISSUE THREAD

**Issue by Arcnor** [Vector35/binaryninja-api#1137]
I have many functions or symbols that are referenced only by symbols in the data section, but when trying to find cross references to those, the Xref list is empty, which means there is no way to see where they are being used from.

There is not even a workaround for this (as far as I know) because I cannot use the text search to find where they are being mentioned (unlike IDA's text search, which you can use to just search on the assembly itself when everything else fails)

**Comment by joshwatson** [comment]
You can use text search. Just escape the hex values like `\x12\x34`

This is also probably a duplicate issue. We’ve been asking for data references for a while 🙂

**Comment by toolCHAINZ** [comment]
I see you're already aware of https://github.com/Vector35/binaryninja-api/issues/303, just putting a link to it anyway for reference.

**Comment by Arcnor** [comment]
@toolCHAINZ I am, but that one only mentions strings from the string view, which is not exactly the same I'm asking for here, I think?

**Comment by toolCHAINZ** [comment]
I assume it's all the same root cause, so figured it's worth linking.

**Comment by Arcnor** [comment]
@joshwatson I don't understand what you mean by escaping. Do you mean escaping the address of the function/symbol I want to find, or the name of the function/symbol, or something else?

**Comment by Arcnor** [comment]
Right, I think you meant the addresses, and probably using "Escaped string" on the search dialog. Yeah, that's a workaround, although a highly inconvenient one of course, but it works, so many thanks again @joshwatson ++!

**Comment by psifertex** [comment]
I know we haven't made public progress on this one but it's high on our internal priority list. We keep a hit list of the top five most impactful bugs and this and related issues are square in it. Don't have an ETA for a fix yet unfortunately but know that we see it too and know it's important.
